scripts/featured

```diff
@@ -364,7 +365,7 @@ class FeatureHandler(object):
     def get_multiasic_feature_instances(self, feature, all_instance=False):
         # Create feature name suffix depending feature is running in host or namespace or in both
         feature_names = (
-            ([feature.name] if feature.has_global_scope or not self.is_multi_npu else []) +
+            ([feature.name] if feature.has_global_scope or all_instance or not self.is_multi_npu else []) +
```

Comment:
why do we need to pass  all_instance, the earlier condition should be good enough ? 
```
if feature.has_global_scope or not self.is_multi_npu
```

Reply:
@judyjoseph : for this use case `if feature.has_global_scope == False or (not self.is_multi_npu == False)`  so we want to disable this service so need to add to the service list using `all_instance`